examples/plugins/analyzer/example_analyzer.cpp

```diff
@@ -150,7 +150,7 @@ class example_plugin final : public virtual analyzer_plugin,
   }
 
   /// Returns the unique name of the plugin.
-  const char* name() const override {
+  std::string_view name() const override {
```

Comment:
Shouldn't these simply return a `std::string`?

Reply:
Considering these are supposed to be string literals I don't think it makes much of a difference in practice; it probably is the safer option, though.

Reply:
Yeah, and it makes it easier to use as a value. It's highly unlikely to be in the critical path and a solid standard library implementation uses SSO for 99% of these short names.